mooncake-store/include/storage_backend.h

```diff
@@ -400,7 +400,8 @@ class BucketIdGenerator {
 
 class BucketStorageBackend {
    public:
-    BucketStorageBackend(const std::string& storage_filepath);
+    BucketStorageBackend(const std::string& storage_filepath,
+                         bool enable_orphan_cleanup = false);
```

Comment:
I’m wondering whether setting the default value to `true` would be better here. After all, cleaning up problematic files is an essential function.